src/command_modules/azure-cli-sql/azure/cli/command_modules/sql/commands.py

```diff
@@ -7,14 +7,31 @@
     get_sql_servers_operations,
     get_sql_firewall_rules_operations,
     get_sql_databases_operations,
-    get_sql_elastic_pools_operations)
+    get_sql_elastic_pools_operations,
+    get_sql_capabilities_operations)
 from azure.cli.core.sdk.util import (
     create_service_adapter,
     ServiceGroup)
 
 custom_path = 'azure.cli.command_modules.sql.custom#{}'
 
 ###############################################
+#                sql capabilities             #
+###############################################
+
+capabilities_operations = create_service_adapter(
+    'azure.mgmt.sql.operations.capabilities_operations',
+    'CapabilitiesOperations')
```

Comment:
The line length limitation is 120 now. You may be able to put them on the same line.